### PR TITLE
Add tests for scaling Alfalfa workers

### PR DESCRIPTION
**Pull request by TShapinsky**

### CODE REVIEW

**Comment by TShapinsky** [comment]
would it make more sense for this to just use pytest-parallel to run existing integration tests at scale?

File: setup.cfg [hunk 1]
```diff
@@ -71,11 +71,12 @@ extras = True
 addopts =
     --cov alfalfa_worker --cov-report term-missing
     --verbose
-    -m "not integration and not fmu and not docker"
+    -m "not integration and not fmu and not docker and not scale"
```
Comment (nllong) [inline setup.cfg:74]: oh my, my de morgan theorem mind is going to hate this in the future.

**Review by nllong — APPROVED** [review]
this is a great test to have! 

1) how many resources can we use from google before they start oomkill'ing our containers?
2) I like the idea of parallel tests, but I am worried that other issues might arise from non-determinism or just general difficulty in debugging the tests.

File: tests/integration/test_broken_models.py [hunk 2]
```diff
@@ -10,18 +10,11 @@ def test_broken_models(broken_model_path):
...
-        else:
-            alfalfa.start(
-                run_id,
-                external_clock=True,
-                start_datetime=0,
-                end_datetime=1000)
+        alfalfa.start(
+            run_id,
+            external_clock=True,
+            start_datetime=datetime.datetime(2019, 1, 2, 0, 2, 0),
```
Comment (anyaelena) [inline tests/integration/test_broken_models.py:16]: this highlights our weirdly specific start time

File: tests/integration/test_single_zone_vav_fmu.py [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
+from tests.integration.conftest import prepare_model
+
 
 @pytest.mark.integration
 class TestSingleZoneVAVFMU(TestCase):
 
     def test_simple_internal_clock(self):
         alfalfa = AlfalfaClient(url='http://localhost')
-        fmu_path = os.path.join(os.path.dirname(__file__), 'models', 'single_zone_vav.fmu')
+        fmu_path = prepare_model('single_zone_vav.fmu')
```
Comment (anyaelena) [inline tests/integration/test_single_zone_vav_fmu.py:16]: nice update.

**Review by anyaelena — APPROVED** [review]

**Pull request merged**